<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabView2DWidget</class>
 <widget class="QWidget" name="tabView2DWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="view2DYZButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>YZ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view2DXZButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>XZ</string>
   </property>
  </widget>
  <widget class="QSlider" name="view2DOverlayOpacitySlider">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>364</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="view2DLayout"/>
  </widget>
  <widget class="QLabel" name="view2DOverlayOpacityLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Overlay Opacity</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="view2DXYButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>XY</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="view2DSliceText">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>330</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>31</width>
     <height>21</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>31</width>
     <height>21</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::NoWrap</enum>
   </property>
   <property name="plainText">
    <string>100</string>
   </property>
  </widget>
  <widget class="QSlider" name="view2DSliceSlider">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>22</width>
     <height>311</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
